batch/batch/front_end/front_end.py

```diff
@@ -1910,30 +1910,62 @@ def plot_job_durations(container_statuses: dict, batch_id: int, job_id: int):
     return json.dumps(fig, cls=plotly.utils.PlotlyJSONEncoder)
 
 
-def plot_resource_usage(resource_usage: Optional[Dict[str, Optional[pd.DataFrame]]]) -> Optional[str]:
+def plot_resource_usage(
+    resource_usage: Optional[Dict[str, Optional[pd.DataFrame]]],
+    memory_limit_bytes: Optional[int],
+    io_storage_limit_bytes: Optional[int],
+    non_io_storage_limit_bytes: Optional[int],
+) -> Optional[str]:
     if resource_usage is None:
         return None
 
+    if io_storage_limit_bytes is not None:
+        if io_storage_limit_bytes == 0:
+            io_storage_title = ''
```

Comment:
In what situation would it be zero? I feel like we should always have some title.

Reply:
This is a placeholder for not having the 6th plot (it's empty) when someone requests `j.storage('0Gi')`. I decided in that case, it was better just to have one plot ("Storage (Container Overlay)") rather than trying to display a graph with 0 storage where `/io` is not really a "drive". Maybe I'm over thinking this.

Reply:
See the Zulip thread for an example.

Reply:
<img width="681" alt="Screen-Shot-2023-01-20-at-2 41 07-PM" src="https://user-images.githubusercontent.com/1693348/214435060-d566b023-59ee-48e9-bfe3-ab7ce90c0495.png">

Reply:
Ah ok, this makes sense seeing the final result in the UI.